src/test/java/cloud/fogbow/ras/core/OrderControllerTest.java

```diff
@@ -1,50 +1,44 @@
 package cloud.fogbow.ras.core;
 
-import cloud.fogbow.common.constants.FogbowConstants;
-import cloud.fogbow.common.exceptions.*;
+import java.util.List;
```

Comment:
general comment: many test method declare `throws` for exceptions that are not actually thrown

Reply:
Have been fixed.